src/generators/app/templates/src/shared/routes/CustomRoute.jsx

```diff
@@ -0,0 +1,133 @@
+/* eslint no-param-reassign: ["error", { "props": false }] */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Route, withRouter, Redirect } from 'react-router';
+import { connect } from 'react-redux';
+
+export const RouteWithStatus = ({ code, children }) => (
+  <Route
+    render={({ staticContext }) => {
+      if (staticContext) {
+        staticContext.status = code;
```

Comment:
there is no `staticContext` on the client, so we need to guard against that here